sqlalchemy_utils/types/email.py

```diff
@@ -4,9 +4,12 @@
 
 
 class EmailType(sa.types.TypeDecorator):
-    impl = sa.Unicode(255)
+    impl = sa.Unicode
     comparator_factory = CaseInsensitiveComparator
 
+    def __init__(self, length=255):
```

Comment:
I would add `*arg, **kwargs` pass them to the next `__init__`. Missing `**kwargs` prevents arguments `collation`, `convert_unicode`,` unicode_error` and `_warn_on_bytestring`. Missing `*args` hinders subclassing.